src/main/java/org/jenkinsci/plugins/ssegateway/sse/EventDispatcher.java

```diff
@@ -293,7 +293,7 @@ private void scheduleRetryQueueProcessing(long delay) {
         }
         if (delay > 0) {
             try {
-                new Timer().schedule( new TimerTask() {
+                new Timer("EventDispatcher.retryProcessor").schedule( new TimerTask() {
```

Comment:
Again, I don't know what this code is trying to do, but seems like it would be better to use a single static `ExecutorService` or `Timer` rather than creating a new `Timer` each time this is called.